notifications/postgres/storage/notification_storage.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package storage
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/lib/pq"
+
+	"github.com/Peripli/service-manager/pkg/types"
+	"github.com/Peripli/service-manager/storage"
+	"github.com/Peripli/service-manager/storage/postgres"
+)
+
+// NotificationStorage storage for getting and listening for notifications
+//go:generate counterfeiter . NotificationStorage
+type NotificationStorage interface {
+	GetNotification(ctx context.Context, id string) (*types.Notification, error)
+
+	// GetLastRevision returns the last received notification revision
+	GetLastRevision(ctx context.Context) (int64, error)
+
+	// NewConnection returns new connection with callback for events
+	NewConnection(eventCallback func(isRunning bool, err error)) NotificationConnection
```

Comment:
that's a bit strange to be part of the interface, plus its in postgres anyway - I would try to remove this method (if its possible ofc)

Reply:
Currently these interfaces are used for testing as storage is lacking some functionality. They should be removed with later PR.